content/posts/annoucing-slight-v0_3.md

```diff
@@ -0,0 +1,226 @@
+---
+title: "Annoucing Slight v0.3"
+description: "Announcing new features and updates on Slight v0.3 release."
+date: 2023-02-25 00:00:00 +0000 UTC
+authorname: "Jiaxiao (Joe) Zhou"
+author: "@jiaxiao_zhou"
+authorlink: "https://twitter.com/jiaiao_zhou"
+tags: ["slight", "spiderlightning", "Wasm", "WASI"]
+---
+
+We are excited to announce the release of `Slight` v0.3, the latest version of our WebAssembly runtime that utilizes SpiderLightning (also known as `wasi-cloud`) capabilities.
+
+You can download the latest release by running the following command if you are using Linux or macOS:
+
+```bash
+/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/deislabs/spiderlightning/main/install.sh)"
+```
+
+If you are using Windows, you can download the latest release by running this command:
+
+```powershell
+iex ((New-Object System.Net.WebClient).DownloadString('https://raw.githubusercontent.com/deislabs/spiderlightning/main/install.ps1'))
```

Comment:
Thanks for providing both bash and powershell compatible instructions! 💯

Reply:
Thank you~